force-app/main/default/triggers/myAccountTrigger.trigger

```diff
@@ -0,0 +1,13 @@
+trigger myAccountTrigger on Account (before insert, after insert) {
+if (Trigger.isBefore && Trigger.IsInsert){
+    AccountTriggerHelper.beforeInsert(Trigger.new);
+}
+
+if (Trigger.isAfter && Trigger.IsInsert){
+    AccountTriggerHelper.beforeInsert(Trigger.new);
+}
+
+if (Trigger.isAfter && Trigger.isInsert) {
+    AccountTriggerHelper.afterInsert(Trigger.new);
+}
+}
```

Comment:
Excellent, but again, mind your indentation of the brackets. Everything should not be left-aligned here.